scripts/us_epa/airpollution/process.py

```diff
@@ -0,0 +1,377 @@
+    "measurementMethod: C:airpollution_emission_trends_tier1->"
+    "Measurement_Method\n"
+    "observationAbout: C:airpollution_emission_trends_tier1->geo_ID\n"
+    "observationDate: C:airpollution_emission_trends_tier1->year\n"
+    "scalingFactor: 1000\n"
+    "unit: Ton\n"
+    "value: C:airpollution_emission_trends_tier1->observation\n")
+
+
+def _add_missing_columns(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Aggregates the columns based on SV
+
+    Args: df (pd.DataFrame): df as the input, to aggregate values
+
+    Returns: df (pd.DataFrame): modified df as output
+    """
+
+    df = df.drop(df[(df['SV'].str.contains('PrescribedFire')) |
+                    (df['SV'].str.contains('Wildfire')) |
+                    (df['SV'].str.contains('Miscellaneous'))].index)
+    # Replacing the columns for grouping as per
+    # StationaryFuelCombustion -    FuelCombustionElectricUtility
+    #                               FuelCombustionIndustrial
+    #                               FuelCombustionOther
+    # IndustrialAndOtherProcesses - ChemicalAndAlliedProductManufacturing
+    #                               MetalsProcessing
+    #                               PetroleumAndRelatedIndustries
+    #                               OtherIndustrialProcesses
+    #                               SolventUtilization
+    #                               StorageAndTransport
+    #                               WasteDisposalAndRecycling
+    # Transportation -              OnRoadVehicles
+    #                               NonRoadEnginesAndVehicles
+    df['SV'] = (df['SV'].str.replace(
+        'FuelCombustionElectricUtility',
+        'StationaryFuelCombustion').str.replace(
+            'FuelCombustionIndustrial', 'StationaryFuelCombustion').str.replace(
+                'FuelCombustionOther', 'StationaryFuelCombustion').str.replace(
+                    'ChemicalAndAlliedProductManufacturing',
+                    'IndustrialAndOtherProcesses').str.replace(
+                        'MetalsProcessing',
+                        'IndustrialAndOtherProcesses').str.replace(
+                            'PetroleumAndRelatedIndustries',
+                            'IndustrialAndOtherProcesses').str.replace(
+                                'OtherIndustrialProcesses',
+                                'IndustrialAndOtherProcesses').str.replace(
+                                    'SolventUtilization',
+                                    'IndustrialAndOtherProcesses').str.replace(
+                                        'StorageAndTransport',
+                                        'IndustrialAndOtherProcesses').str.
+                replace('WasteDisposalAndRecycling',
+                        'IndustrialAndOtherProcesses').str.replace(
+                            'OnRoadVehicles', 'Transportation').str.replace(
+                                'NonRoadEnginesAndVehicles', 'Transportation'))
```

Comment:
Please change this to dict of kv's to find and replace. Run within for loop to replace values.
e.g., for k, v in d: df['SV'].str.replace(k,v)

Reply:
We were using the dictionary method in ASR earier, however Jhangir asked us to use it inline only.